File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)

{  
    float dollar_change;
    int cent_change;
    int coins = 0;

    do
    {
        dollar_change = get_float("Enter the change amount: ");
    }
    while (dollar_change <= 0.00);

    //Convert Dollar Change to Cent Change
    cent_change = round(dollar_change * 100);

    while (cent_change >= 25)
    {
        cent_change = cent_change - 25;
        coins++;
    }

    while (cent_change >= 10)
    {
        cent_change = cent_change - 10;
        coins++;
    }

    while (cent_change >= 5)
    {
        cent_change = cent_change - 5;
        coins++;
    }

    while (cent_change >= 1)
    {
        cent_change = cent_change - 1;
        coins++;
    }
    printf("You will recieve a total of: %i coins \n", coins);
}

File: cash.py
from cs50 import get_float

## original dollar change amount input 
dollar = get_float("Change owed: ")
if dollar < 0:
    print ("Enter non-negative value")
    dollar = get_float("Change owed: ")

## dollars converted into coin change amount    
total_change = dollar * 100

## Coins needed intiialized to zero 
coins = 0

while total_change > 0: 
    if total_change >= 25:
        total_change = total_change - 25
        coins += 1
    elif total_change >= 10: 
        total_change = total_change - 10
        coins += 1
    elif total_change >= 5:
        total_change = total_change - 5
        coins += 1
    else:
        total_change = total_change - 1
        coins += 1
    
print(coins)